gateway/1.5/modules/ROOT/pages/_partials/task-config-logs-third-party.adoc

```diff
@@ -224,6 +236,51 @@ spec:
 
 //end::http-configuration-example[]
 
+//tag::dynatrace-configuration-example[]
+
+[[dynatrace-configuration-example]]
+== Dynatrace Configuration Example
+
+You can configure Flex Gateway to send formatted runtime and access logs to a Dynatrace environment `HTTP` output. As an example, the following file contains the necessary configurations:
+
+[source,yaml]
+----
+apiVersion: gateway.mulesoft.com/v1alpha1
+kind: Configuration
+metadata:
+  name: dynatrace-config
+spec:
+  logging:
+    outputs:
+    - name: dynatrace
+      type: http
+      parameters:
+        host: <environmentID>.live.dynatrace.com        
+        Port: "443"
+        allow_duplicated_headers: "false"        
+        header: 
+          - Content-Type application/json; charset=utf-8
+          - Authorization Api-Token <accessToken>
+        URI: /api/v2/logs/ingest
+        Format: json
+        json_date_key: timestamp
+        json_date_format: iso8601
+        tls: "on"
+        tls.verify: "off"
+    runtimeLogs:
+      logLevel: info
+      outputs:
+        - dynatrace
```

Comment:
Do we have to add some comment regarding multiple Configuration Files? it happens to me that I have 2, one defining default output with runtimeLogs, and accessLogs, then the output is added to the final config but not the merge of these 2 elements. It take me some time to discover why